ansible/templates/minigraph_link_meta.j2

```diff
@@ -23,6 +23,30 @@
           </a:Properties>
         <a:Key>{{ dual_tor_facts['positions']['lower'] }}:{{ tunnel }};{{ dual_tor_facts['positions']['upper'] }}:{{ tunnel }}</a:Key>
       </a:LinkMetadata>
+{% endfor %}
```

Comment:
hi roman,
please remove the changes from template files as it can be picked from my changes

Reply:
> hi roman, please remove the changes from template files as it can be picked from my changes

Done